deep_quoridor/src/metrics.py

```diff
@@ -12,11 +12,20 @@ class Metrics:
     Computes metrics for an agent by making it play against other agents.
     """
 
-    def __init__(self, board_size: int, max_walls: int, benchmarks: list[str | Agent] = []):
+    def __init__(
+        self,
+        board_size: int,
+        max_walls: int,
+        benchmarks: list[str | Agent] = [],
+        benchmarks_t: int = 10,
+        max_steps=200,
+    ):
         self.board_size = board_size
         self.max_walls = max_walls
         self.stored_elos = {}
         self.benchmarks = benchmarks
+        self.benchmarks_t = benchmarks_t
+        self.max_steps = max_steps
```

Comment:
This ones were hardcoded before, and I made it so that you can specify from the arguments now

Reply:
Ah, this is great! I've been wishing that the number of games in the benchmarks was configurable!